### ISSUE TITLE
[BUG] Windows icon generation error

### ISSUE THREAD

**Issue by yaymalaga** [mrrhak/icons_launcher#5]
### :information_source: Info
Latest version: 1.1.7

### :speech_balloon: Description
The following error is returned:
```
FileSystemException: Cannot open file, path = 'windows/Runner/resources/app_icon.ico' (OS Error: No such file or directory, errno = 2)
```
On my system (linux), the correct path would be windows/runner/.....

### :scroll: Pubspec.yaml
dev_dependencies:
  icons_launcher: ^1.1.7

flutter_icons:
  image_path: 'assets/icons/my.png'
  adaptive_icon_round: 'assets/icons/my.png'
  adaptive_icon_background: 'assets/icons/my.png'
  adaptive_icon_foreground: 'assets/icons/my.png' 
  remove_alpha_ios: true
  remove_alpha_macos: true
  ios: true
  android: true
  macos: true
  windows: true
  linux: true
  web: true

**Comment by mrrhak** [comment]
After you run the command  `icons_launcher:main`. Did you get icon file name `app_icon.ico` in the directory `windows/runner/resources/app_icon.ico`?

Sorry I don't have a Linux system to verify it but I test with the Windows system and it looks fine. This [example](https://github.com/mrrhak/icons_launcher/tree/master/example) I already tested with platform (android, ios, macos, windows and web)

**Comment by yaymalaga** [comment]
The thing is that the folder inside windows is not called 'Runner' but 'runner'. I don't know if this is platform specific, which it shouldn't. 

In the official Flutter Gallery example, the runner folder is named in lowercase: https://github.com/flutter/gallery/tree/master/windows/runner

**Comment by mrrhak** [comment]
Ohh, thank you I will update it to lowercase in the next version. I test with the windows system it's not restricted case-sensitive but Linux strongly restricts case-sensitive.

**Comment by mrrhak** [comment]
This issue has been fixed on [v1.1.8](https://pub.dev/packages/icons_launcher). Thanks
